modules/bullet/SCsub

```diff
@@ -0,0 +1,191 @@
+        , "BulletDynamics/ConstraintSolver/btSolve2LinearConstraint.cpp"
+        , "BulletDynamics/ConstraintSolver/btTypedConstraint.cpp"
+        , "BulletDynamics/ConstraintSolver/btUniversalConstraint.cpp"
+        , "BulletDynamics/Dynamics/btDiscreteDynamicsWorld.cpp"
+        , "BulletDynamics/Dynamics/btDiscreteDynamicsWorldMt.cpp"
+        , "BulletDynamics/Dynamics/btSimulationIslandManagerMt.cpp"
+        , "BulletDynamics/Dynamics/btRigidBody.cpp"
+        , "BulletDynamics/Dynamics/btSimpleDynamicsWorld.cpp"
+        #, "BulletDynamics/Dynamics/Bullet-C-API.cpp"
+        , "BulletDynamics/Vehicle/btRaycastVehicle.cpp"
+        , "BulletDynamics/Vehicle/btWheelInfo.cpp"
+        , "BulletDynamics/Featherstone/btMultiBody.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyConstraintSolver.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyJointLimitConstraint.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyConstraint.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyPoint2Point.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyFixedConstraint.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodySliderConstraint.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyJointMotor.cpp"
+        , "BulletDynamics/Featherstone/btMultiBodyGearConstraint.cpp"
+        , "BulletDynamics/MLCPSolvers/btDantzigLCP.cpp"
+        , "BulletDynamics/MLCPSolvers/btMLCPSolver.cpp"
+        , "BulletDynamics/MLCPSolvers/btLemkeAlgorithm.cpp"
+
+        # BulletInverseDynamics
+        , "BulletInverseDynamics/IDMath.cpp"
+        , "BulletInverseDynamics/MultiBodyTree.cpp"
+        , "BulletInverseDynamics/details/MultiBodyTreeInitCache.cpp"
+        , "BulletInverseDynamics/details/MultiBodyTreeImpl.cpp"
+
+        # BulletSoftBody
+        , "BulletSoftBody/btSoftBody.cpp"
+        , "BulletSoftBody/btSoftBodyConcaveCollisionAlgorithm.cpp"
+        , "BulletSoftBody/btSoftBodyHelpers.cpp"
+        , "BulletSoftBody/btSoftBodyRigidBodyCollisionConfiguration.cpp"
+        , "BulletSoftBody/btSoftRigidCollisionAlgorithm.cpp"
+        , "BulletSoftBody/btSoftRigidDynamicsWorld.cpp"
+        , "BulletSoftBody/btSoftMultiBodyDynamicsWorld.cpp"
+        , "BulletSoftBody/btSoftSoftCollisionAlgorithm.cpp"
+        , "BulletSoftBody/btDefaultSoftBodySolver.cpp"
+
+        # clew
+        , "clew/clew.c"
+
+        # LinearMath
+        , "LinearMath/btAlignedAllocator.cpp"
+        , "LinearMath/btConvexHull.cpp"
+        , "LinearMath/btConvexHullComputer.cpp"
+        , "LinearMath/btGeometryUtil.cpp"
+        , "LinearMath/btPolarDecomposition.cpp"
+        , "LinearMath/btQuickprof.cpp"
+        , "LinearMath/btSerializer.cpp"
+        , "LinearMath/btSerializer64.cpp"
+        , "LinearMath/btThreads.cpp"
+        , "LinearMath/btVector3.cpp"
+    ]
+
+thirdparty_dir = "#thirdparty/Bullet/"
```

Comment:
All thirdparty directories in `thirdparty` use lower case names, think you could change that one to `thirdparty/bullet/` too?

Just do:
```
git mv thirdparty/Bullet thirdparty/bullet
git add thirdparty/bullet
// edit modules/bullet/SCsub accordingly
git add modules/bullet/SCsub
git commit --amend
```